### PR TITLE
(DOCSP-20564)+(DOCSP-20596) KMIP Update

### PR DESCRIPTION
**Pull request by biniona-mongodb**
## Pull Request Info

Update language and diagram in KMIP guide. Remove build errors.

### Issue JIRA link:

https://jira.mongodb.org/browse/DOCSP-20564

(by removing build error this also completes)
https://jira.mongodb.org/browse/DOCSP-20596

### Snooty build log:

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=61fad4ef382d7fe4ee17f7e4

### Docs staging link (requires sign-in on MongoDB Corp SSO):

https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCSP-20564-KMIP-Update/security/client-side-field-level-encryption-local-key-to-kms/#configure-your-kmip-provider

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [ ] Are all the links working? (Issues with right hand TOC that seem to be unrelated as[ it's present on the currently deployed site](https://docs.mongodb.com/drivers/security/client-side-field-level-encryption-local-key-to-kms/))
- [x] Are the staging and workerpool job links in the PR description updated?

### CODE REVIEW

File: source/includes/steps-fle-convert-to-a-remote-master-key-kmip.yaml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
-  To connect a MongoDB driver client to your KMIP KMS, you must configure your KMS
-  such that it accepts your client's TLS certificate.
+  To connect a MongoDB driver client to your KMIP provider, you must configure
+  your KMIP provider such that it accepts your client's TLS certificate.
 
-  Consult the documentation for your KMIP KMS for information on how
-  to accept your client certificate with your specific KMS.
+  Consult the documentation for your KMIP provider for information on how
+  to accept your client certificate with your specific KMIP provider.
```
Comment (terakilobyte) [inline source/includes/steps-fle-convert-to-a-remote-master-key-kmip.yaml:9]: I know it's existing text, but now that I read it again it's very redundant.

```suggestion
  to accept your client certificate.
```

File: source/security/client-side-field-level-encryption-local-key-to-kms.txt [hunk 2]
```diff
@@ -93,8 +93,8 @@ provider:
 
       .. include:: /includes/steps/fle-convert-to-a-remote-master-key-gcp.rst
 
-   .. tab:: KMIP KMS
-      :tabid: kmip-kms
+   .. tab:: KMIP Provider
+      :tabid: kmip-privoder
```
Comment (terakilobyte) [inline source/security/client-side-field-level-encryption-local-key-to-kms.txt:97]: The world isn't ready for a **privoder**.

```suggestion
      :tabid: kmip-provider
```

**Review by terakilobyte — APPROVED** [review]
LGTM  with suggested corrections.

File: source/includes/steps-fle-convert-to-a-remote-master-key-kmip.yaml (hunk 1 above)
Comment (biniona-mongodb) [reply source/includes/steps-fle-convert-to-a-remote-master-key-kmip.yaml:9]: Yup agree this is clear without the repetition

File: source/security/client-side-field-level-encryption-local-key-to-kms.txt (hunk 2 above)
Comment (biniona-mongodb) [reply source/security/client-side-field-level-encryption-local-key-to-kms.txt:97]: Just googled "privoder" and it means "driver" in [Russian](https://translate.google.com/?source=dict-chrome-ex&sl=ru&tl=en&q=privoder), which makes this mistake almost correct.

JK changing to provider.

**Comment by biniona-mongodb** [comment]
Cynthia Braund approved this PR over Slack.

**Pull request merged**